### ISSUE TITLE
500 vs 404 error

### ISSUE THREAD

**Issue by bobbysebolao** [fac-17/SaVaGe#81]
500 indicates a server error. 404 indicates the resource isn't found, which is what you're describing on line 19 of handlers.js (inside your staticAssets function). If one of the requested files isn't found, it probably wouldn't be due to a server error, just an incorrect URL.

**Comment by crianonim** [comment]
You are right, I'll see if I can find a way to differentiate between the two situations! The reason we put 500 because it is internal I/O error, could be a file that doesn't exist (404) or actual I/O error (disk error, etc 500 code) .  Thanks for pointing it out!
